upf/upf_pfcp_server.h

```diff
@@ -110,6 +110,7 @@ typedef struct
 typedef struct
 {
   uword session_idx;
+  u64 cp_seid;
```

Comment:
the increase of the data structure by 8 byte (from 21 to 29) … does it change the memory impact of the running instance for 10k 100k 200k sessions in a noticeable way?

Reply:
This structure is only used when briefly queueing Session Report Requests, so as not to send them right away while handling the user plane packets, and these requests aren't generated often enough to keep more than a few dozen or so in the queue. So I think this will likely have no impact.